src/components/TopBar/CallButton.vue

```diff
@@ -33,21 +33,19 @@
 			:disabled="startCallButtonDisabled || loading || blockCalls"
 			:type="startCallButtonType"
 			@click="handleClick">
-			<template #icon>
-				<span class="icon"
-					:class="startCallIcon" />
-			</template>
+			<Video slot="icon"
```

Comment:
![Bildschirmfoto vom 2022-05-18 08-14-03](https://user-images.githubusercontent.com/213943/168969799-b261ba44-2d6b-4b65-848f-a2b674687622.png)

@juliushaertl what's up with this phpstorm note?

Reply:
Makes sense to migrate then I guess, but so far it does not seem to be part of our eslint config, so maybe it is a local check of PHPstorm or some plugin there?

https://eslint.vuejs.org/rules/no-deprecated-slot-attribute.html
> This rule is included in all of "plugin:vue/vue3-essential", "plugin:vue/vue3-strongly-recommended" and "plugin:vue/vue3-recommended". 

And we don't pull in any of those https://github.com/nextcloud/eslint-config/blob/master/index.js#L20

Reply:
The recommended syntax is to use a `template` component in conjunction with with the `v-slot` directive (or its shorthand `# `, see https://v2.vuejs.org/v2/guide/components-slots.html#Named-Slots

So like
```vue
<template #icon>
	<Video />
</template>
```
That saves a bit of effort if you ever migrate to vue3, because the old syntax is not supported there anymore.